reservoir.install

```diff
@@ -29,8 +29,11 @@ function reservoir_install() {
 
   // Generate initial, temporary keys, for a better onboarding experience.
   if (function_exists('openssl_pkey_new')) {
-    $oauth_settings = \Drupal::configFactory()->getEditable('simple_oauth.settings');
     list($private_key, $public_key) = _reservoir_generate_keypair();
+  }
+
```

Comment:
Let's remove this blank line, so that it's clear the comment on line 30 also applies to this.